### DISCUSSION TITLE
nuxtj : Uncaught (in promise) RangeError: Adding different instances of a keyed plugin (plugin$2)

### DISCUSSION THREAD

**Discussion by williamtse** [ueberdosis/tiptap#953] [General Questions]
when i use tiptap in nuxtjs project , i got this bug:
Uncaught (in promise) RangeError: Adding different instances of a keyed plugin (plugin$2)

**Comment by hanspagel** [comment]
Thanks for reporting! This should be fixed with the new version! Would you mind to try again?
